examples/c/example.c

```diff
@@ -266,81 +261,81 @@ AranyaError add_sync_peers(Team *t, AranyaSyncPeerConfig *cfg) {
     return ARANYA_ERROR_SUCCESS;
 }
 
-// Run the example.
+// Runs the C example program.
 AranyaError run(Team *t) {
     AranyaError err;
 
     // initialize logging.
     printf("initializing logging\r\n");
-    err = aranya_init_logging();
-    EXPECT("error initializing logging", err);
+    EXPECT(aranya_init_logging(), "error initializing logging");
 
     // initialize the Aranya team.
     printf("initializing team\r\n");
-    err = init_team(t);
-    EXPECT("error initializing team", err);
+    EXPECT(init_team(t), "error initializing team");
+
+    Client *owner = &t->clients.owner;
+    Client *admin = &t->clients.admin;
+    Client *operator= & t->clients.operator;
```

Comment:
nit: looks like there's an extra space before the `&` here:
```suggestion
    Client *operator= &t->clients.operator;
```